staff/marc-uson/spotify-template/index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <link rel="stylesheet" href="style/style.css">
+    <link rel="icon" href="https://open.scdn.co/static/images/favicon.png ">
+    <title>Browse - Featured</title>
+</head>
+<body>
+    
+    <header class="header-mbl">
+        <img class="logo" src="images/Spotify_Logo_white.png" alt="logo">
+        <span class="button-mbl"><a href="">VIEW ON SPOTIFY</a></span>
+    </header>
+
+    <aside class="aside">
+        <img class="logo" src="images/Spotify_Logo_white.png" alt="logo">
+        <section class="aside__buttons">
+            <a class="aside__buttons__home" href="">
```

Comment:
@Markuson BEM naming has to be improved! Once you have used underscore, next children should be added with `-`. Naming would look like: `aside__buttons-home`-

Reply:
OK, done!